app/src/main/res/values/strings.xml

```diff
@@ -42,7 +42,7 @@
     <string name="common_go_to_swap">Go to swap</string>
     <string name="common_sell_all">Sell all</string>
     <string name="common_sorry">Sorry</string>
-    <string name="common_less_then_minimum">Less then 0.01</string>
+    <string name="common_less_then_minimum">Less than 0.01</string>
```

Comment:
will you fix key as well? `common_less_then_minimum` -> `common_less_than_minimum`